tests/contrib/funders/test_funders_schema.py

```diff
@@ -10,16 +10,42 @@
 import pytest
 from marshmallow import ValidationError
 
-from invenio_vocabularies.contrib.funders.schema import FunderRelationSchema
+from invenio_vocabularies.contrib.funders.schema import FunderRelationSchema, \
+    FunderSchema
 
 
```

Comment:
moderate: it ties up with one of the comments above, but there should be a test to check the actual `id`/`scheme` value format.